src/build/content/server.ts

```diff
@@ -260,9 +260,9 @@ export const copyNextDependencies = async (ctx: PluginContext): Promise<void> =>
   await tracer.withActiveSpan('copyNextDependencies', async () => {
     const entries = await readdir(ctx.standaloneDir)
     const promises: Promise<void>[] = entries.map(async (entry) => {
-      // copy all except the package.json and distDir (.next) folder as this is handled in a separate function
+      // copy all except the distDir (.next) folder as this is handled in a separate function
       // this will include the node_modules folder as well
-      if (entry === 'package.json' || entry === ctx.nextDistDir) {
+      if (entry === ctx.nextDistDir) {
```

Comment:
we are no longer writing top level package.json (we write it just for runtime modules, so it no longer is top-level), hence this copies top-level package.json provided by Next.js (which is really just site's package.json) so it will "inherit" `type` from user's site